core/src/test/scala/com/salesforce/op/filters/FeatureDistributionTest.scala

```diff
@@ -238,4 +241,21 @@ class FeatureDistributionTest extends FlatSpec with PassengerSparkFixtureTest wi
     intercept[IllegalArgumentException](fd1.jsDivergence(fd1.copy(name = "boo"))) should have message
       "requirement failed: Name must match to compare or combine feature distributions: A != boo"
   }
+
+  it should "not serialize cardEstimate field" in {
+    val fd1 = FeatureDistribution("A", None, 10, 1, Array(1, 4, 0, 0, 6),
+      Array.empty, Some(Moments(1.0)), Some(TextStats(Map("foo" -> 1, "bar" ->2))),
+      FeatureDistributionType.Scoring)
+    FeatureDistribution.toJson(Seq(fd1)) shouldNot include ("cardEstimate")
+
+    val json =
```

Comment:
You already computed this string on Line 249, you can store it in a variable.

Reply:
that json doesn't contains cardEstimate obj, which ignored while deserialize, i construct another example with it, which should be tested that fromJson can still deserialize this obj without ignoring it.

Reply:
you can generate it too:
```
val jsonWithCardEstimate = Serialization.write(fd1)(DefaultFormats +
      EnumEntrySerializer.json4s[FeatureDistributionType](FeatureDistributionType))
```